SpatialGDK/Documentation/content/tutorials/offloading-tutorial/offloading-gameplay-changes.md

```diff
@@ -77,13 +75,15 @@ void UEquippedComponent::EndPlay(const EEndPlayReason::Type EndPlayReason)
 
 ### Conclusion
 
-By using offloading, you can scale your CPU heavy tasks by splitting them to run on instances of different server-workers. You don’t need to worry about the challenges involved with migrating entities across worker boundaries.
+By using offloading, you can scale your CPU heavy tasks by splitting them to run on instances of different server-workers. You don’t need to worry about the challenges involved with migrating entities across worker boundaries. 
+
 
 Meanwhile, offloading introduces a set of constraints that you need to take care of.  When you set up your feature using offloading properly, you can have significant scaling opportunities.
 
 <br/>------------<br/>
+_2019-08-12 Page updated with limited editorial review_
```

Comment:
@claireimprobable  - FYI - Added the update status to the page changelogs where there are changes more than the odd word.
Thanks for updating. Will give it a proper look when I do the JIRA ticket. (Next task for me.)